app/assets/javascripts/items.js

```diff
@@ -2,9 +2,10 @@
 // All this logic will automatically be available in application.js.
 
 $(document).ready(function() {
-
+console.log('setting up click handlers')
```

Comment:
You might want to remove this in a later branch/commit to take out debugging code. Doesn't have to be done in this PR